voicenger/settings/base.py

```diff
@@ -149,18 +152,33 @@
     "http://127.0.0.1:3000",
     ]
 
-SIMPLE_JWT = {
-    'ACCESS_TOKEN_LIFETIME': timedelta(minutes=5),
-    'REFRESH_TOKEN_LIFETIME': timedelta(days=1),
-    'ROTATE_REFRESH_TOKENS': False,
-    'BLACKLIST_AFTER_ROTATION': True,
-    'ALGORITHM': 'HS256',
-    'SIGNING_KEY': SECRET_KEY,
-    'VERIFYING_KEY': None,
-    'AUTH_HEADER_TYPES': ('Bearer',),
-    'AUTH_TOKEN_CLASSES': ('rest_framework_simplejwt.tokens.AccessToken',),
-    'TOKEN_TYPE_CLAIM': 'token_type',
-}
+# SIMPLE_JWT = {
+#     'ACCESS_TOKEN_LIFETIME': timedelta(minutes=5),
+#     'REFRESH_TOKEN_LIFETIME': timedelta(days=1),
+#     'ROTATE_REFRESH_TOKENS': False,
+#     'BLACKLIST_AFTER_ROTATION': True,
+#     'ALGORITHM': 'HS256',
+#     'SIGNING_KEY': SECRET_KEY,
+#     'VERIFYING_KEY': None,
+#     'AUTH_HEADER_TYPES': ('Bearer',),
+#     'AUTH_TOKEN_CLASSES': ('rest_framework_simplejwt.tokens.AccessToken',),
+#     'TOKEN_TYPE_CLAIM': 'token_type',
+# }
+
+# SIMPLE_JWT = {
+#     'ALGORITHM': 'RS256',
+#     'VERIFYING_KEY': 'KbBNx2MGrxhkmzqn0LMkp6W5oXomLtgc',
+#     'AUDIENCE': 'https://dev-m4yp61l880yrkjnn.us.auth0.com/api/v2/',
+#     'ISSUER': 'https://dev-m4yp61l880yrkjnn.us.auth0.com/',
```

Comment:
Is that how it's supposed to be in the public view?

Reply:
There is no secret data in this line, these are public keys that are already visible during authorization, but I agree it is better to remove them.

Reply:
Perhaps it makes sense to put them in an .env file?